draft-ietf-shmoo-online-meeting.md

```diff
@@ -120,143 +175,150 @@ one of those regions.
 | In-Person A | In-Person A | Online A Night | see below |
 | Online A Night | Online B Night | Online C Night | A Night |
 
-Basically this table follows two guidelines:
-1) When ever a fully online meeting follows an in-person meeting, the online meeting time is used that disadvantages most the participants of the time zone where the in-person meeting was held.
-2) If multiple fully online meetings follow each other, the time zone selection should be rotated based on the most recent time zones that the in-person meetings were held in.
+This table follows two basic guidelines:
+1) When ever a fully online meeting follows an in-person meeting, the online
+meeting time is used that disadvantages most the participants of the time zone
+where the in-person meeting was held.
+2) If multiple fully online meetings follow each other, the time zone selection
+should be rotated based on the most recent time zones that the in-person
+meetings were held in.
 
 The final case occurs in the rare event that back-to-back in-person plenaries
 occur in the same region. In this case, find the most recent meeting that was
 neither in 'A' (if in person) nor in 'A' night (if remote). If this meeting
 was in-person in region 'B', then the next meeting should be in 'B' Night. If it
 was remote in 'B' Night, the next meeting should be in 'C' Night.
 
-To initialize this algorithm, IETF 112 is considered as an 'Asia Night'
-remote meeting, and IETF 111 is a 'Europe Night' remote meeting.
-
 ## Number of Days and Total Hours per Day
 
-Online meetings have converged to run over 5 days with 6-hour meeting days, roughly.
-Only the administrative plenary, which concludes with multiple open mic sessions, is not necessarily time-bounded.
+By 2021, online meetings were consistently over 5 days with roughly 6-hour
+meeting days. The administrative plenary, which concludes with multiple open mic
+sessions, would sometimes exceed this limit.
 
-Based on the experience so far, 6 hours of online meetings, with two 30 minutes breaks,
-appears to be potentially a natural limit of what is handleable for most participants.
-Respectively, the meeting survey after IETF 109 has indicated a high satisfaction with the distribution of sessions
-over 5 days but only a medium satisfaction with the
-overall length of each day [https://www.ietf.org/blog/ietf108-survey-results-informed-planning/].
+6 hours of online meetings, with two 30-minute breaks, was a compromise
+between the physical limits of attending an online meeting in an inconvenient
+time zone, and the demand for many sessions with a manageable number of
+conflicts. The IETF 109 feedback {{109-SURVEY}} indicated broad satisfaction
+with this compromise.
```

Comment:
Not sure that reflects the survey accurately. It think it's also important to mention that people are not willing to extend only meetings to the weekend, so there are not much options.